Wikipedia/Code/BaseTableViewCell.swift

```diff
@@ -0,0 +1,8 @@
+import UIKit
+
+@objc(WMFBaseTableViewCell)
+class BaseTableViewCell: UITableViewCell {
```

Comment:
You could remove this class and use `UITableViewCell` directly (unless there are plans to put more functionality here?)